dbinspector/src/main/java/im/dino/dbinspector/fragments/TableFragment.java

```diff
@@ -19,243 +19,240 @@
 import android.widget.TableLayout;
 import android.widget.TableRow;
 import android.widget.TextView;
-
-import java.io.File;
-import java.util.List;
-
 import im.dino.dbinspector.adapters.TablePageAdapter;
 import im.dino.dbview.R;
+import java.io.File;
+import java.util.List;
 
 /**
  * Created by dino on 24/02/14.
  */
 @TargetApi(Build.VERSION_CODES.HONEYCOMB)
-public class TableFragment extends Fragment implements ActionBar.OnNavigationListener {
-
-    private static final String KEY_DATABASE = "database_name";
-
-    private static final String KEY_TABLE = "table_name";
-
-    private static final String KEY_SHOWING_CONTENT = "showing_content";
+public class TableFragment
+		extends Fragment
+		implements ActionBar.OnNavigationListener {
 
-    private static final String KEY_PAGE = "current_page";
+	private static final String KEY_DATABASE = "database_name";
```

Comment:
@PrashamTrivedi the same comment as for other files applies here. It's really hard to see shat you did when everything is marked as changed.